src/components/Modal/Modal.tsx

```diff
@@ -121,7 +121,7 @@ export const Modal: React.FunctionComponent<ModalProps> & {
       if (iframe && iframe.contentWindow) {
         try {
           setIframeHeight(iframe.contentWindow.document.body.scrollHeight);
-        } catch {
+        } catch (err) {
```

Comment:
We typically use something like:

```suggestion
        } catch (_) {
```

or 
```suggestion
        } catch (_error) {
```

To show that we're not using a var that needs to be declared.